Code/Framework/AzFramework/AzFramework/Entity/GameEntityContextComponent.cpp

```diff
@@ -294,13 +294,12 @@ namespace AzFramework
                     {
                         SpawnableEntitiesDefinition* spawnableEntitiesInterface = SpawnableEntitiesInterface::Get();
                         AZ_Assert(spawnableEntitiesInterface != nullptr, "SpawnableEntitiesInterface is not found.");
-                        spawnableEntitiesInterface->RetrieveEntitySpawnTicket(
-                            currentEntity->GetSpawnTicketId(),
-                            [spawnableEntitiesInterface, currentEntity](EntitySpawnTicket* entitySpawnTicket)
+                        spawnableEntitiesInterface->RetrieveTicket(currentEntity->GetSpawnTicketId(),
```

Comment:
very minor concern: The class is EntitySpawnTicket, the SpawnableEntitiesInterface only uses the word "Ticket" in its function names (which is honestly fine by me given the context), but then entity has "GetSpawnTicketId" which is kind of a hybrid between the two. I'd either use `GetEntitySpawnTicketId` or `GetTicketId` instead.

Reply:
I'd go for `GetEntitySpawnTicketId` as the entity is external to the Spawnable system, so the extra context is warranted.